sensor/common/deduper/unchanged.go

```diff
@@ -7,26 +7,26 @@ import (
 	"github.com/stackrox/rox/pkg/sync"
 )
 
-// ObservationSet holds IDs that were seen during the initial sensor sync.
-type ObservationSet struct {
+// ClosableSet holds IDs that were seen during the initial sensor sync.
```

Comment:
```suggestion
// ClosableSet is a set of strings. Items can be added to the set as long as the set is open. Closing the set ignores all future `Add` operations but allows to fetch already stored items. 
```

Reply:
Done. Just changed to also add info about it being concurrency-safe.